src/Tribe/Importer/Options.php

```diff
@@ -124,4 +137,54 @@ public static function get_possible_stati() {
 
 		return apply_filters( 'tribe_import_possible_stati', $stati );
 	}
+
+	/**
+	 *  Return Encoding Status
+	 *
+	 * @param string $type importer type
+	 *
+	 * @return mixed|void
+	 */
+	public static function get_default_encoding_status( $type = 'csv' ) {
+
+		$options = self::getOption( 'imported_encoding_status', array( $type => 'encode' ) );
+
+		if ( ! isset( $options[ $type ] ) ) {
+			$options[ $type ] = 'encode';
+		}
+
+		if ( ! isset( $options[ $type ] ) ) {
+			/**
+			 * Filter Encode Status if none saved
+			 *
+			 * @since 4.2
+			 *
+			 * @param string $type importer type
```

Comment:
We need two `@param` tags here - one for the value being filtered (`'encode'`) and one for additional param `$type`.